### PR TITLE
Updated frontend Code. It now uses vuetify, and localstorage. Added l…

### PR DESCRIPTION
**Pull request by cf2080**
# Issue/Ticket

CLOSES #XX

# Description
Updated frontend Code. It now uses vuetify, and local storage. Added link to reset password page

## Type of change
Chore

Label your PR accordingly.
manually tested

### CODE REVIEW

File: Backend/DataAccessLayer/Repositories/ApiKeyRepository.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ namespace DataAccessLayer.Repositories
 {
     public class ApiKeyRepository
     {
-        /// <summary>
+        /// <summary>
```
Comment (julianpoy) [inline Backend/DataAccessLayer/Repositories/ApiKeyRepository.cs]: This should not be changed.

File: Frontend/src/const.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
-const port = 61348;
-const route = `http://sso.julianjp.com:8080/api`;
+const dev_port = 61348;
+const dev_route = `http://localhost:${dev_port}/api`;
 
 export default {
-    API_PORT: port,
-    API_ROUTE: route
+    DEV_PORT: dev_port,
+    DEV_ROUTE: dev_route
```
Comment (julianpoy) [inline Frontend/src/const.js]: Don't revert this file. These need to come from env variables, so either make that change or leave as-is for merge.

File: Frontend/src/main.js [hunk 3]
```diff
@@ -8,5 +8,6 @@ Vue.config.productionTip = false
 
 new Vue({
   router,
+  axios,
```
Comment (julianpoy) [inline Frontend/src/main.js]: Why this change?

File: Frontend/src/services/request.js [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
+let apiURL = localStorage.getItem('base') || `${window.location.protocol}//${window.location.hostname}:${dev_const.DEV_PORT}/api`;
 
 /* eslint-disable */
 let token = localStorage.getItem('token');
 
 const register = registrationData => {
   return axios.post(`${apiURL}/users/register`, registrationData)
     .then(response => {
-      token = response.data.token;
+      token = response.data.data.token;
```
Comment (julianpoy) [inline Frontend/src/services/request.js]: None of these changes should be present. These are all reversions of the changes I made in my branch.

File: Frontend/src/views/Register.vue [hunk 5]
```diff
@@ -199,7 +187,7 @@ export default {
       }).catch(err => {
         switch(err.response.status) {
           case 401:
-            this.error = err.response.data;
+            this.error = "Your password has been cracked before. Please use a more secure password.";
             break;
           case 406:
             this.error = "Your email address does not follow a recognized format.";
```
Comment (julianpoy) [inline Frontend/src/views/Register.vue]: Again, you're changing registration. This should not be changed.

**Review by julianpoy — CHANGES_REQUESTED** [review]
See my comments. Also, the comments I made in your other PR apply here.

**Comment by alfredovargasjr** [comment]
@cf2080 make this also has the latest changes

File: Frontend/src/views/Register.vue [hunk 6]
```diff
@@ -190,7 +182,6 @@ export default {
         securityQ3Answer: this.securityQ3Answer  
       }).then(() => {
         const params = new URLSearchParams(window.location.search)
-
```
Comment (julianpoy) [inline Frontend/src/views/Register.vue]: This file should not be changed in this PR.

**Review by julianpoy — APPROVED** [review]

**Pull request merged**